cmd/pak.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2018 the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/GoogleCloudPlatform/gcp-service-broker/pkg/brokerpak"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	pakCmd := &cobra.Command{
+		Use:   "pak",
+		Short: "interact with user-defined service definition bundles",
+		Long: `Lets you create, validate, and view service definition bundles.
+
+A service definition bundle is a zip file containing all the elements needed
+to define and run a custom service.
+
+Bundles include source code (for legal compliance), service definitions, and
+Terraform/provider binaries for multiple platforms. They give you a contained
+way to deploy new services to existing brokers or augment the broker to fit
+your needs.
+
+To start building a pack, create a new directory and within it run init:
+
+	mkdir my-pak && cd my-pak
+	gcp-service-broker pak init
```

Comment:
nit: The build command allows you to specify a directory, where as init does not. I wonder if it makes sense for consistency sake to have init to accept an optional path arg or else default to the current directory.

Reply:
++